src/components/PokemonCart.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+export default function PokemonCart({ cart, removePokemonFromCart }) {
+  return (
+    <>
+      <h4>Pokemon Cart</h4>
+      <p>Total: ${cart.reduce((total, pokemon) => total + pokemon.price, 0)}</p>
+      <button className="btn btn-primary btn-block" type="button" data-toggle="collapse" data-target="#cartPokemon" aria-expanded="false" aria-controls="cartPokemon">
+    List of pokemons in cart
+  </button>
+      <div className="collapse" id="cartPokemon">
+        {cart.map(pokemon => (
```

Comment:
here if you only need the name you can destructure it from pokemon: `({ name })`